_events/2023-11-08-intro-workshop.md

```diff
@@ -0,0 +1,60 @@
+---
+short_title: CHTC Introduction Workshop
+title: "CHTC Introduction Workshop"
+published: true
+
+excerpt: |
+	Get hands-on experience with large-scale computing by attending this half-day workshop introducing CHTC services. 
+
+start_date: 2023-11-08
+end_date: 2023-11-08
+location: "UW-Madison Campus, Room 1240 of the Computer Sciences Building"
+image: "/images/FirstDayClass_AD21_0269.jpg"
```

Comment:
I think the image you chose is fine, but if you wanted one with a more CHTC feel, you could choose one from Monday's HTCondor week: https://www.dropbox.com/sh/9sx0o5omgwo6sje/AABG467qSHdO2w3hjNALcxFea/mon?dl=0&subfolder_nav_tracking=1

for example: https://www.dropbox.com/sh/9sx0o5omgwo6sje/AAAfsDrtdiNnVo3E0qi2HODMa/mon/DSC04105.jpg?dl=0 is very similar to what you chose, but it's HTC